### DISCUSSION TITLE
Destination file and path names, timezone

### DISCUSSION THREAD

**Discussion by Amphiboly** [ep1cman/unifi-protect-backup#26] [Ideas]
All is working well, thank you. This is an enhancement request.

I see that the destination file and path names are based on the UTC date expansion of the event start time. The result for me (in EST5EDT) is that many events are placed in a directory with the next day's date (relative to the local time).

![image](https://user-images.githubusercontent.com/10307457/157556673-b50b50af-2497-497c-968e-1b5213b619e6.png)

Is there a way to use the local time to fashion the file and directory names?

-- 
Rik

**Comment by ep1cman** [comment]
This is definitely a bug, I never encountered it as I happen to be in the UK (GMT). The unifi api returns all timestamps with the timezone set to UTC, when saving to file this should be changed to the local time zone for sure. I will create an issue from this discussion and post updates on there as I investigate and fix this.

**Comment by ep1cman** [comment]
Are you able to test the current "main" branch to see if this fixes the issue for you before I do a release
